frontend/ui/src/components/Stepper/StepChild.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from "react";
+import { Step, StepContent, StepLabel, Typography } from "@mui/material";
+import { StepperButtons } from "./StepperButtons";
+
+export const StepChild: FC<{
+  step: {
+    label: string;
+    description: string;
+    content: JSX.Element;
+  };
+  handleNext: () => void;
+  handleBack: () => void;
+  index: number;
+  stepLength: number;
+}> = ({ step, handleNext, handleBack, index, stepLength }) => {
+  return (
+    <>
+      <StepLabel
+        optional={
+          index === 2 ? (
+            <Typography variant="caption">Last step</Typography>
+          ) : null
+        }
```

Comment:
what does it represent index === 2?

Reply:
Oh it doesn't have to be like that, it represent the last step, but it's wrong that it's hardcoded, i will change it now, thanks for notice it!